python/test/t_HSICEstimatorTargetSensitivity_std.py

```diff
@@ -0,0 +1,90 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
```

Comment:
for next time: we try to write tests with "import openturns as ot" the same way we do in examples

Reply:
Done